internal/tracing/conn_windows.go

```diff
@@ -0,0 +1,14 @@
+package tracing
+
+import (
+	"context"
+	"net"
+	"time"
+
+	"github.com/Microsoft/go-winio"
+)
+
+func DialInMemory(ctx context.Context, addr string) (net.Conn, error) {
+	timeout := 1 * time.Second
+	return winio.DialPipeContext(ctx, addr, &timeout)
```

Comment:
Looks like CI complains about the timeout attribute (cf : https://github.com/docker/compose/actions/runs/4877301442/jobs/8737538726#step:6:322)